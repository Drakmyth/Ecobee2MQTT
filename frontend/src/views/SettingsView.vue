<script setup lang="ts">
import EcobeeConfiguration from "@/components/EcobeeConfiguration.vue";
import MQTTConfiguration from "@/components/MQTTConfiguration.vue";
import BridgeConfiguration from "@/components/BridgeConfiguration.vue";
</script>

<template>
    <div class="settings">
        <Suspense>
            <EcobeeConfiguration />
        </Suspense>
        <MQTTConfiguration />
        <BridgeConfiguration />
    </div>
</template>

<style>
</style>
