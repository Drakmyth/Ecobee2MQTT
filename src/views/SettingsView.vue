<script setup lang="ts">
import EcobeeConfiguration from "../components/EcobeeConfiguration.vue";
import MQTTConfiguration from "../components/MQTTConfiguration.vue";
import BridgeConfiguration from "../components/BridgeConfiguration.vue";
</script>

<template>
    <div class="settings">
        <EcobeeConfiguration />
        <MQTTConfiguration />
        <BridgeConfiguration />
    </div>
</template>

<style>
</style>
