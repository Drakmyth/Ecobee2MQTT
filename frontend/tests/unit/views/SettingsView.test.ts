import { shallowMount } from '@vue/test-utils';
import SettingsView from '@/views/SettingsView.vue';

describe('SettingsView.vue', () => {
    test('renders correctly', () => {
        const wrapper = shallowMount(SettingsView);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
