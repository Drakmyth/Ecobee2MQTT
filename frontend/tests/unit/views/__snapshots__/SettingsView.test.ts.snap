// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsView.vue renders correctly 1`] = `
"<div class=\\"settings\\">
  <suspense-stub></suspense-stub>
  <!---->
  <!---->
</div>"
`;
